shepherd-client/src/test/java/com/devsda/platform/shepherdclient/impl/PayLoad.java

```diff
@@ -0,0 +1,50 @@
+package com.devsda.platform.shepherdclient.impl;
+
+import java.util.Date;
+
+public class PayLoad {
```

Comment:
This makes document  rigid. Payload should be Map<String, Object>. Wanted to talk more on this. Lets discuss offline.

Reply:
removed, this was for testing purpose